contracts/validators/CredentialAtomicQueryMTPValidator.sol

```diff
@@ -3,11 +3,24 @@ pragma solidity 0.8.16;
 
 import {CredentialAtomicQueryValidator} from "./CredentialAtomicQueryValidator.sol";
 import {IVerifier} from "../interfaces/IVerifier.sol";
-import {IState} from "../interfaces/IState.sol";
 
 contract CredentialAtomicQueryMTPValidator is CredentialAtomicQueryValidator {
     string internal constant CIRCUIT_ID = "credentialAtomicQueryMTPV2OnChain";
 
+    struct PublicSignals {
+        uint256 merklized;
+        uint256 userID;
+        uint256 circuitQueryHash;
+        uint256 requestID;
+        uint256 challenge;
+        uint256 gistRoot;
+        uint256 issuerID;
+        uint256 issuerClaimIdenState;
+        uint256 isRevocationChecked;
```

Comment:
Let's rename isRevocationChecked to skipClaimRevocationCheck as in the protocol messages.
And also not to confuse developers to set in payload isRevocationChecked = !skipClaimRevocationCheck in the query builder and do according changes in proof check.

require(signals.isRevocationChecked == 0 &&  !skipClaimRevocationCheck , "Revocation check is expected");

Reply:
I named this fields exactly as they are in circuits: https://github.com/iden3/circuits/blob/master/circuits/onchain/credentialAtomicQueryMTPOnChain.circom#L106